source/mongofiles.txt

```diff
@@ -14,9 +14,7 @@
 .. |tool-binary| replace:: :binary:`~bin.mongofiles`
 .. |mongodb-aws-example| replace:: :ref:`mongofiles-example-connect-using-aws-iam`
 
-.. only:: (not man)
-
-   .. binary:: mongofiles
+.. binary:: mongofiles
```

Comment:
• Do we still need this? Other binary ref pages do not have this. I ask because: I'm unsure myself!

Reply:
Sometimes y'all are using `program` and sometimes :binary: in your links, so I just left them both in - if you want to standardize on one, then we can pull the other one out.

Reply:
We should standardize, but that's on us imo. Let's leave as you have here.